paper/paper.md

```diff
@@ -40,29 +40,40 @@ bibliography: paper.bib
 # Summary
 
 TrixiParticles.jl, part of the Trixi Framework [@schlottkelakemper2020trixi], is an innovative Julia-based open-source software designed for particle-based multiphysics simulations.
-In this sense, we distinguish two approaches: Particles are either treated as physical particles or as mathematical interpolation points.
-The former case refers to methods that model separate, discrete particles with rotational degrees of freedom such as the Discrete Element Method (DEM) proposed by [@Cundall:1979].
-Whereas the latter case refers to the smoothed particle hydrodynamics (SPH) which is a numerical discretization method for solving continuum mechanic problems.
-SPH was originally developed by [@Monaghan:1977] to simulate astrophysical applications and is now widely used to simulate computational fluid dynamics, structural mechanics and even heat conduction problems.
+It aims to handle complex geometries and specialized applications, such as computational fluid dynamics (CFD) and structural dynamics,
+by offering a versatile platform for particle-based methods.
+To increase flexibility, TrixiParticles.jl facilitates the easy addition of new particle systems and their interactions.
+Coupling of different particle systems is very user friendly, thus multiphysics simulations such as fluid-structure interaction (FSI) can be set up very quickly.
+This is further complemented by allowing simulations to be configured directly with Julia code.
+This feature not only simplifies the integration of custom functionalities but also promotes rapid prototyping,
+establishing TrixiParticles.jl as a versatile and adaptable tool for advancing multiphysics simulations in various scientific and engineering fields.
```

Comment:
I like this new summary! Only this half sentence sounds like too much advertising.